src/main/java/dao/AirportDaoImpl.java

```diff
@@ -0,0 +1,107 @@
+package dao;
+
+import entities.Airport;
+import services.ConnectionFactory;
+
+import java.sql.*;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Implementation of airport DAO
+ * @author klysov
+ */
+public class AirportDaoImpl implements AirportDao{
+    @Override
+    public Airport extractAirportFromResultSet(ResultSet rs) throws SQLException {
+        Airport airport = new Airport();
+        airport.setIdAirport( rs.getInt("idAirport") );
+        airport.setName( rs.getString("Name") );
+        return airport;
+    }
+
+    @Override
+    public Set<Airport> getAllAirports() {
+        Connection connection = ConnectionFactory.getConnection();
+        try {
+            Statement stmt = connection.createStatement();
+            ResultSet rs = stmt.executeQuery("SELECT * FROM Airport");
+            Set<Airport> airports = new HashSet<>();
+            while(rs.next())
+            {
+                Airport airport = extractAirportFromResultSet(rs);
+                airports.add(airport);
+            }
+            return airports;
+        } catch (SQLException ex) {
+            ex.printStackTrace();
```

Comment:
why do you catch it here? printStackTrace is not sulution of exception it jest for logging. I think you can add exception throws to all dao methods and process them somewere else